header-footer-grid/Core/Builder/Abstract_Builder.php

```diff
@@ -1107,6 +1132,10 @@ private function add_row_style( $row_index, $css_array = array() ) {
 			return $this->add_legacy_row_styles( $css_array, $row_index );
 		}
 
+		if ( get_theme_mod( 'neve_pro_global_header_settings_advanced_style', true ) === false && $this->get_id() === 'header' && $row_index !== 'sidebar' ) {
+			return $css_array;
+		}
+
```

Comment:
The return here is a bit too early 👀 We still need some of the rules below this point when using a global background. Here are the exact rules that need to be excluded in that context 🙈 

https://github.com/Codeinwp/neve/blob/d5a01bb1f72393431b3d82025b99110d07d8f0c5/header-footer-grid/Core/Builder/Abstract_Builder.php#L1156-L1227